cheroot/makefile.py

```diff
@@ -384,4 +376,27 @@ def readline(self, size=-1):
                 return ''.join(buffers)
 
 
-MakeFile = MakeFile_PY2 if six.PY2 else MakeFile_PY3
+if six.PY3:
+    class StreamReader(io.BufferedReader):
+        """Socket stream reader."""
+
+        def __init__(self, sock, mode='r', bufsize=io.DEFAULT_BUFFER_SIZE):
+            """Initialize socket stream reader."""
+            super().__init__(socket.SocketIO(sock, mode), bufsize)
```

Comment:
_(This is always evaluated under Python 3, so no need to use Python 2 compatible `super()` syntax)_